Shared/Samples/Select features in scene layer/SelectFeaturesInSceneLayerView.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SelectFeaturesInSceneLayerView: View {
+    /// The point on the screen where the user tapped.
+    @State private var tapPoint: CGPoint?
+    
+    /// The scene layer that is added on top of the scene
+    @State private var sceneLayer = {
+        ArcGISSceneLayer(url: .brestBuildingService)
+    }()
+    
+    /// A scene with topographic basemap and a 3D buildings layer.
+    @State private var scene: ArcGIS.Scene = {
+        // Creates a scene and sets an initial viewpoint.
+        let scene = Scene(basemapStyle: .arcGISTopographic)
+        let point = Point(x: -4.49779, y: 48.38282, z: 40, spatialReference: .wgs84)
+        // set an initial camera location
+        let camera = Camera(location: point, heading: 41.65, pitch: 71.2, roll: 0)
+        // set initial viewpoint to camera position at point
+        scene.initialViewpoint = Viewpoint(boundingGeometry: point, camera: camera)
+        // Creates a surface and adds an elevation source.
+        let surface = Surface()
+        surface.addElevationSource(ArcGISTiledElevationSource(url: .worldElevationService))
+        // Sets the surface to the scene's base surface.
+        scene.baseSurface = surface
+        return scene
+    }()
+    
+    /// Add feature layer to scene in initialization
+    init() {
+        scene.addOperationalLayer(sceneLayer)
+    }
```

Comment:
Sorry I didn't make myself clear. What you have here is totally fine, no need to change. 
Another approach can be adding a computed property to get the layer from the array instead of holding it as a state property.

```swift
    private var sceneLayer: ArcGISSceneLayer {
        scene.operationalLayers.first as! ArcGISSceneLayer
    }
```